packages/checkout-storefront/src/providers/BillingSameAsShippingProvider.tsx

```diff
@@ -12,7 +12,7 @@ interface BillingSameAsShippingContextConsumerProps {
 export const [useBillingSameAsShipping, Provider] =
   createSafeContext<BillingSameAsShippingContextConsumerProps>();
 
-export const BillingSameAsShippingProvider: React.FC<PropsWithChildren<{}>> = ({ children }) => {
+export const BillingSameAsShippingProvider: React.FC<{ children: ReactNode }> = ({ children }) => {
```

Comment:
We have type `children` defined in globalTypes

Reply:
Yes but it defines `children` as either `ReactNode` or `ReactNode[]`. That's not what we want in most cases.

Reply:
Yeah but it's that we want in this particular component 😅